server/validate_signature.go

```diff
@@ -0,0 +1,25 @@
+package server
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/hex"
+	"errors"
+	"fmt"
+)
+
+func validateSignature(recievedHash []string, bodyBuffer []byte, key string) error {
```

Comment:
Thank you for your work on this! I think there is a typo in recieve. Maybe a test would be nice for this validate function, what do you think?

Reply:
Fully agree!

Reply:
Fully agree this one, thanks for catching typo!

Reply:
I've make func public, added comment and tests :+1: